### PR TITLE
OSU MPI Microbenchmarks for MVAPICH MPI testcase

### PR DESCRIPTION
**Pull request by ekmalkan**
Added new test case for OSU Benchmarks for MVAPICH MPI.
Ran the test locally and received PASS for multiple trials.

[LISAv2 Test Results Summary]
Test Run On           : 05/12/2020 20:21:30
ARM Image Under Test  : OpenLogic : CentOS : 7.7 : latest
Initial Kernel Version: 3.10.0-1062.12.1.el7.x86_64
Final Kernel Version  : 3.10.0-1062.12.1.el7.x86_64
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:22

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes)
-------------------------------------------------------------------------------------------------------------------------------------------
    1 INFINIBAND           INFINIBAND-MVAPICH-MPI-OMB-2VM                                                    PASS                18.19
        InfiniBand-Verification-1-FirstBoot : eth0 IP : PASS
        InfiniBand-Verification-1-FirstBoot : IBV_PINGPONG : PASS
        InfiniBand-Verification-1-FirstBoot : PingPong Intranode : PASS
        InfiniBand-Verification-1-FirstBoot : IMB-MPI1 : PASS
        InfiniBand-Verification-1-FirstBoot : OMB-P2P : PASS
        InfiniBand-Verification-2-Reboot : eth0 IP : PASS
        InfiniBand-Verification-2-Reboot : IBV_PINGPONG : PASS
        InfiniBand-Verification-2-Reboot : PingPong Intranode : PASS
        InfiniBand-Verification-2-Reboot : IMB-MPI1 : PASS
        InfiniBand-Verification-2-Reboot : OMB-P2P : PASS

### CODE REVIEW

**Comment by ekmalkan** [comment]
[Azure-RU81-TestLogs.zip](https://github.com/LIS/LISAv2/files/4618423/Azure-RU81-TestLogs.zip)

Test Logs attached with test Results

File: cgmanifest.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (juhlee-microsoft) [inline cgmanifest.json]: I am not sure if I can add this file into public Github repo.

File: Testscripts/Linux/TestRDMA_MultiVM.sh [hunk 2]
```diff
@@ -715,6 +767,7 @@ function Main() {
 			mpi_run_path=$(find / -name mpirun | grep intel64)
 		;;
 		mvapich)
+			omb_p2p_path="/root/mvapich2-2.3/osu_benchmarks/mpi/pt2pt/"
```
Comment (juhlee-microsoft) [inline Testscripts/Linux/TestRDMA_MultiVM.sh]: If we have a newer version to OSU_MPI_BENCHMARK_DOWNLOAD, then how can you manage this path? Your path has version number.

File: Testscripts/Linux/TestRDMA_MultiVM.sh [hunk 3]
```diff
@@ -169,6 +174,52 @@ function Run_IMB_MPI1() {
 	fi
 }
 
+function Run_OMB_P2P() {
+	total_attempts=$(seq 1 1 $omb_p2p_tests_iterations)
+	omb_p2p_tests_array=($omb_p2p_tests)
+	for attempt in $total_attempts; do
+		LogMsg "OMB P2P test iteration $attempt for $mpi_type- Running."
+		case "$mpi_type" in
+			ibm)
```
Comment (juhlee-microsoft) [inline Testscripts/Linux/TestRDMA_MultiVM.sh]: please combine all into a single condition.

**Review by juhlee-microsoft — CHANGES_REQUESTED** [review]

File: cgmanifest.json (hunk 1 above)
Comment (ekmalkan) [reply cgmanifest.json]: Removed in revision

File: Testscripts/Linux/TestRDMA_MultiVM.sh (hunk 2 above)
Comment (ekmalkan) [reply Testscripts/Linux/TestRDMA_MultiVM.sh]: Modified in revision

File: Testscripts/Linux/TestRDMA_MultiVM.sh (hunk 3 above)
Comment (ekmalkan) [reply Testscripts/Linux/TestRDMA_MultiVM.sh]: done

**Review by juhlee-microsoft — APPROVED** [review]
It looks good. Wait for legal confirmation of licensing.

File: Testscripts/Linux/TestRDMA_MultiVM.sh [hunk 4]
```diff
@@ -45,6 +46,10 @@ function Collect_Logs() {
 	# Create zip file to avoid downloading thousands of files in some cases
 	zip -r /root/IMB_AllTestLogs.zip /root/IMB-*
 	rm -rf /root/IMB-*
+	if [[ $benchmark_type == "OMB" ]]; then
+	zip -r /root/OMB_AllTestLogs.zip /root/OMB-*
```
Comment (LiliDeng) [inline Testscripts/Linux/TestRDMA_MultiVM.sh:50]: indent

File: Testscripts/Linux/TestRDMA_MultiVM.sh [hunk 5]
```diff
@@ -730,8 +774,12 @@ function Main() {
 	LogMsg "IMB-NBC Path: $imb_nbc_path"
 	LogMsg "IMB-P2P Path: $imb_p2p_path"
 	LogMsg "IMB-IO Path: $imb_io_path"
+	LogMsg "OMB Path: $omb_path"
 
 	# Run all benchmarks
+	if [[ $benchmark_type == "OMB" ]]; then
+	Run_OMB_P2P
```
Comment (LiliDeng) [inline Testscripts/Linux/TestRDMA_MultiVM.sh:781]: indent

**Pull request merged**